docs/content/exporting/http/fastapi-gunicorn.md

```diff
@@ -19,7 +19,7 @@ metrics_app = make_asgi_app()
 app.mount("/metrics", metrics_app)
 ```
 
-For Multiprocessing support, use this modified code snippet. Full multiprocessing instructions are provided [here](https://github.com/prometheus/client_python#multiprocess-mode-eg-gunicorn).
+For Multiprocessing support, use this modified code snippet. Full multiprocessing instructions are provided [here](https://prometheus.github.io/client_python/multiprocess/).
```

Comment:
```suggestion
For Multiprocessing support, use this modified code snippet. Full multiprocessing instructions are provided [here]({{< ref "/multiprocess" >}}).
```
For local dev/if we change where the docs are hosted this will be more robust.